python/py-xarray/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-xarray
-version             0.16.2
+version             0.18.0
```

Comment:
this version support Python 3.7+ so you will need to restrict the PY36 subport to the latest supported version.

Reply:
no worries, I see that you made the changes already. Once you push the changes to your own repo, it will update this PR automatically - so no need to do a new PR.